src/infercnvpy/io/_genepos.py

```diff
@@ -39,8 +39,8 @@ def genomic_position_from_gtf(
         If True, add the annotations directly to adata, otherwise return a dataframe.
     """
     gtf = gtfparse.read_gtf(
-        gtf_file, usecols=["seqname", "feature", "start", "end", "gene_id", "gene_name"]
-    ).to_pandas()
```

Comment:
What exactly was the bug you were facing here? Shouldn't the new and the old version be equivalent? 

Just asking because I remember some back-and-forth on this line in the past, because some versions of gtfparse are incompatible with some polars version.

Reply:
Hi @grst, I realise that the PR I was working on was a previous version of inferCNVpy that did not have the `.to_pandas`, therefore I was getting an error relating to indexes. However you are correct, setting `.to_pandas` and `result_type= pandas` are equivalent. I will revert